hphp/hack/test/quickfixes/non-existent-type-const-names.php

```diff
@@ -0,0 +1,9 @@
+<?hh
+
+class MyClass {
+  const type TFooBar = int;
+  const type TFoo = int;
+  public function foo(this::TFooBaz $_): MyClass::TFooBaz {
```

Comment:
I suggest you remove the parameter here so it's easier to see what you're testing.

```suggestion
  public function foo(): MyClass::TFooBaz {
```